src/main/java/com/axis/jenkins/plugins/eiffel/eiffelbroadcaster/Util.java

```diff
@@ -95,6 +96,22 @@ public static void publishEvent(EiffelEvent event) {
         }
     }
 
+    /**
+     * Splits the input string into lines, removes leading and trailing whitespace, and adds non-empty
+     * lines to a collection.
+     *
+     * @param s the line to split
+     * @param collection the collection to add to
+     */
+    public static void linesToCollection(String s, final Collection<String> collection) {
```

Comment:
Optional: Not a super big fan of void methods manipulating one of the parameters. Would prefer this method to return the collection and let the caller handle the union. If not, at least naming the method clearer: addLinesToCollection().

Reply:
Yeah. This might've made sense in some earlier incarnation of the patch but when looking at it now it's not great. I've changed it to this instead:

```Java
public static List<String> getLinesInString(String s)
```